elixir_phoenix/bin/setup

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+echo installing phoenix dependencies
+mix local.rebar >/dev/null
+mix deps.get >/dev/null
+echo running phoenix server
+mix phoenix.server >/dev/null 2>&1 &
+mkdir -p tmp
+echo $! > tmp/server.pid
+
+until $(curl --output /dev/null --silent --head --fail http://localhost:6969/api/test); do
```

Comment:
nice. i should adopt this for the other servers, instead of assuming itll take less than 1s to boot